lib/form/paragraph.flow

```diff
@@ -681,9 +681,12 @@ breakTextFragment(
 						PositionAllowBreakAfter() : {
 							toInspectingElementIfNecessary(indexing,
 								indexing + keyPosRelative,
-								makeSpace(getCharAt(text, keyPos),
-									style
-								)
+								// in case of CJKLang we don't show the break symbol
```

Comment:
Extract the knowledge about the language's specifics from paragraph.flow into separate file related to languages and use it's functions here.
The function could for example returns special data type with all required field to be used here

Reply:
@ykuzmenko73 addressed